tests/png/bug00295.c

```diff
@@ -0,0 +1,45 @@
+/**
+ * Regression test for <https://github.com/libgd/libgd/issues/295>.
+ *
+ * We're testing that reading a PNG which might trigger a libpng warning, does
+ * either trigger a GD_WARNING with the respective message or no error at all.
+ */
+
+
+#include "gd.h"
+#include "gd_errors.h"
+#include "gdtest.h"
+#include <string.h>
+
+
+#define EXPECTED_MSG "gd-png: libpng warning: iCCP: known incorrect sRGB profile"

```

Comment:
Thanks, good catch. Thinking about it, *this* test shouldn't be concerned about the message at all, and not even about GD_WARNING, but rather that there's no GD_ERROR.